### ISSUE TITLE
Rewrap in an assembly file mangles all the text.

### ISSUE THREAD

**Issue by Lokathor** [stkb/Rewrap#186]
In assembly files using GNU as, text such as this:

```arm
  @ Grab both IE and IF into a single register then bit_and them together. The value that's left is the set of flags that we need to respond to.
  ldr r1, [r0, #0x200]     @ r1 = IE in low, IF in high
  and r1, r1, r1, lsr #16  @ r1 = r1 & (r1 >> 16), aka IE & IF
```

When re-wrapped turns into a mess:
```arm
  @ Grab both IE and IF into a single register then bit_and them together. The
  value that's left is the set of flags that we need to respond to. ldr r1, [r0,
  #0x200]     @ r1 = IE in low, IF in high and r1, r1, r1, lsr #16  @ r1 = r1 &
  (r1 >> 16), aka IE & IF
```

In this case, only the comment starting with `@` should be wrapped. Other lines are code lines, and should be left alone.

Expected output:
```arm
  @ Grab both IE and IF into a single register then bit_and them together. The
  @ value that's left is the set of flags that we need to respond to.
  ldr r1, [r0, #0x200]     @ r1 = IE in low, IF in high
  and r1, r1, r1, lsr #16  @ r1 = r1 & (r1 >> 16), aka IE & IF
```

**Comment by stkb** [comment]
Hi, do you use an extension in VSCode to work with assembly files (a language extension, for syntax coloring etc) and if so which one?

**Comment by Lokathor** [comment]
Yes, I use `ARM` by dan-c-underwood, <https://github.com/dan-c-underwood/vscode-arm>

**Comment by stkb** [comment]
Ok in the just-released version of Rewrap (1.10.0) it should now work (Rewrap detects from the extension that it's a code file with `@` and `/* ... */' comment markers).

The the end-of-line comments won't get any wrapping however (tracked in #46)
